### PR TITLE
Updating sponser and gallery

### PR DESCRIPTION
**Pull request by ayushinigam**
Sponser and gallery sections added

### CODE REVIEW

**Comment by rgabs** [comment]
@ayushinigam Please take a pull from develop and create a separate Jade file for your view and include it in index.jade

**Comment by ayushinigam** [comment]
It's done. Please review the PR now.

File: frontEnd/app/index.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  <script src="/bower_components/bootstrap-sass/assets/javascripts/bootstrap/tab.js"></script>
+  <!-- endbuild -->
+
+  <!-- build:js scripts/main.js -->
+  <script src="scripts/main.js"></script>
+  <!-- endbuild -->
+  
+  
+</body>
+
```
Comment (rgabs) [inline frontEnd/app/index.html]: Please remove index.html from app folder as index.jade is there

File: frontEnd/app/index.jade [hunk 2]
```diff
@@ -46,7 +47,7 @@ body.body-tag
 		ga('send', 'pageview');
 	// build:js scripts/vendor.js
 	// bower:js
-	script(src='/bower_components/jquery/dist/jquery.js')
+	//script(src='/bower_components/jquery/dist/jquery.js')
```
Comment (rgabs) [inline frontEnd/app/index.jade]: Why did you remove jquery?
Please comment the code from main.js

File: frontEnd/app/styles/thirdView.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  background: url(../images/images.jpg);
+  height: 30vh;
+}
+.gallery-image4{
+  background: url(../images/images.jpg);
+  height: 30vh;
+}
+.gallery-image5{
+  background: url(../images/images.jpg);
+  height: 30vh;
```
Comment (rgabs) [inline frontEnd/app/styles/thirdView.scss]: As you can see, the height property is being repeated.
Please make a separate class (gallery-height or something) and assign that class to the element.
